### DISCUSSION TITLE
On ram reading of termux

### DISCUSSION THREAD

**Discussion by 3611895075** [termux/termux-app#3503] [Q&A]
The ram read by my termux is different from that of my mobile phone.Sometimes it is contrary to the available memory of the mobile phone, and sometimes the error is very large.![Screenshot_20230710_010931.jpg](https://github.com/termux/termux-app/assets/127006434/c9171624-b675-495a-9b97-61f39151b73b)

![Screenshot_20230710_010859.jpg](https://github.com/termux/termux-app/assets/127006434/97878c8b-126d-4b52-bcab-e27e664b058c)

**Comment by twaik** [comment]
ZeroTermux is not related to termux. Open issue in appropriate repository.
